packages/icon/src/react/index.js

```diff
@@ -9,11 +9,11 @@ import * as vars from '../vars/index.js'
 
 const style = {
   icon: ({ color, size }) =>
-    css({
-      ...stylesheet['.psds-icon'],
-      ...stylesheet[`.psds-icon--size-${size}`],
-      ...stylesheet[`.psds-icon--color-${color}`]
-    })
+    css(
```

Comment:
did you test this out. i tested out using compose but this might work as well 🤷‍♂